pyproject.toml

```diff
@@ -4,7 +4,92 @@ requires = [
   "setuptools>=50",
   "setuptools_scm>=6",
   "wheel>=0.34",
-]  # ought to mirror 'requirements/build.txt'
+]
+
+[project]
+name = "xrtpy"
+readme = "README.md"
+keywords = ["solar physics"]
+description = "For analyzing data from the X-Ray Telescope (XRT) on the Hinode spacecraft."
+license = {file = "LICENSE"}
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: BSD License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Topic :: Scientific/Engineering :: Astronomy",
+  "Topic :: Scientific/Engineering :: Physics",
+]
+# SPEC 0 recommends that packages in the scientific pythoniverse support
+# versions of Python that have been released in the last ≤36 months
+requires-python = ">=3.9"
+dynamic = ["version"]
+authors = [
+  {name = "Joy Velasquez", email = "joy.velasquez@cfa.harvard.edu"},
+  {name = "Jonathan Slavin", email = "jslavin@cfa.harvard.edu"},
+  {name = "Nick Murpy", email="namurphy@cfa.harvard.edu"},
+  {name = "Will Barnes"},
+  {name = "Stuart Mumford"},
+]
+
+dependencies = [
+  "astropy >= 5.1",
+  "cached-property >= 1.5.2",
+  "matplotlib >= 3.5.0",
+  "numpy >= 1.24.0",
+  "requests >= 2.28.0",
+  "scikit-image >= 0.19.0",
+  "scipy >= 1.8.0",
+  "sunpy[map] >= 4.0.0",
+]
```

Comment:
I updated the dependencies here to be a bit more consistent with [SPEC 0](https://scientific-python.org/specs/spec-0000/) (i.e., having the minimum required versions of dependencies be ∼24 months old).